app/src/androidTest/java/com/github/geohunt/app/ui/components/activehunts/ActiveHuntsTest.kt

```diff
@@ -7,17 +7,20 @@ import androidx.core.content.ContextCompat
 import androidx.core.graphics.drawable.toBitmap
 import androidx.test.platform.app.InstrumentationRegistry
 import com.github.geohunt.app.R
+import com.github.geohunt.app.authentication.Authenticator
```

Comment:
This import, as well as the imports on lines 12, 17, 18, 20, 23 are unused and should be optimized.